recipe/meta.yaml

```
@@ -38,6 +38,9 @@ outputs:
   - name: libxc-c
     build:
       string: {{ proc_type }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      ignore_run_exports_from:
+        # Ignore `cudatoolkit` dependency in CUDA 11 builds
+        - {{ compiler("cuda") }}     # [(cuda_compiler_version or "").startswith("11")]
```

Looks like there is a similar warning from the C++ compiler [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=844840&view=logs&jobId=b07d827c-9ab2-5a9b-a8c1-91faecadc29a&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412 ):

```
WARNING (libxc-c): dso library package conda-forge/linux-64::libstdcxx-ng==13.2.0=h7e041cc_3 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

So we may want to do the same thing with the C++ compiler

```suggestion
        # Ignore `cudatoolkit` dependency in CUDA 11 builds
        - {{ compiler("cxx") }}      # [(cuda_compiler_version or "").startswith("11")]
        - {{ compiler("cuda") }}     # [(cuda_compiler_version or "").startswith("11")]
```